go.mod

```diff
@@ -1,56 +1,54 @@
-module github.com/ochinchina/supervisord
+module github.com/cyralinc/supervisord
 
-go 1.17
+go 1.22
 
 require (
-	github.com/gorilla/mux v1.8.0
-	github.com/gorilla/rpc v1.2.0
-	github.com/jessevdk/go-flags v1.5.0
-	github.com/kardianos/service v1.2.1
+	github.com/cyralinc/supervisord/config v0.0.0-20230902082938-c2cae38b7454
+	github.com/cyralinc/supervisord/events v0.0.0-20230902082938-c2cae38b7454
+	github.com/cyralinc/supervisord/faults v0.0.0-20230902082938-c2cae38b7454
+	github.com/cyralinc/supervisord/logger v0.0.0-20230902082938-c2cae38b7454
+	github.com/cyralinc/supervisord/process v0.0.0-20230902082938-c2cae38b7454
+	github.com/cyralinc/supervisord/signals v0.0.0-20230902082938-c2cae38b7454
+	github.com/cyralinc/supervisord/types v0.0.0-20230902082938-c2cae38b7454
+	github.com/cyralinc/supervisord/util v0.0.0-20230902082938-c2cae38b7454
+	github.com/cyralinc/supervisord/xmlrpcclient v0.0.0-20230902082938-c2cae38b7454
+	github.com/gorilla/mux v1.8.1
+	github.com/gorilla/rpc v1.2.1
+	github.com/jessevdk/go-flags v1.6.1
+	github.com/kardianos/service v1.2.2
 	github.com/ochinchina/go-daemon v0.1.5
 	github.com/ochinchina/go-ini v1.0.1
 	github.com/ochinchina/go-reaper v0.0.0-20181016012355-6b11389e79fc
 	github.com/ochinchina/gorilla-xmlrpc v0.0.0-20171012055324-ecf2fe693a2c
-	github.com/ochinchina/supervisord/config v0.0.0-20220721095143-c2527852d28f
-	github.com/ochinchina/supervisord/events v0.0.0-20220721095143-c2527852d28f
-	github.com/ochinchina/supervisord/faults v0.0.0-20220721095143-c2527852d28f
-	github.com/ochinchina/supervisord/logger v0.0.0-20220721095143-c2527852d28f
-	github.com/ochinchina/supervisord/process v0.0.0-20220520055329-8fdf6b62a44f
-	github.com/ochinchina/supervisord/signals v0.0.0-20220721095143-c2527852d28f
-	github.com/ochinchina/supervisord/types v0.0.0-20220520055329-8fdf6b62a44f
-	github.com/ochinchina/supervisord/util v0.0.0-20220721095143-c2527852d28f
-	github.com/ochinchina/supervisord/xmlrpcclient v0.0.0-20220520055329-8fdf6b62a44f
```

Comment:
I'd suggest leaving these dependencies as they are and simply adding replace directives such as:

github.com/ochinchina/supervisord/config => ./config

This would also avoid the need for us to do version maintenance of the submodules and always the version from the current commit itself will be chosen.